Lib/test/test_syntax.py

```diff
@@ -1877,6 +1877,23 @@
 Traceback (most recent call last):
 SyntaxError: cannot assign to f-string expression here. Maybe you meant '==' instead of '='?
 
+>>> ft'abc'
+Traceback (most recent call last):
+SyntaxError: can't use 'f' and 't' string prefixes at the same time
+
+>>> x = 1
```

Comment:
I don't think this is needed, since the syntax error will fire before the name error below.